### PR TITLE
DDP 6586 | Add address tab for PanCan

### PR DESCRIPTION
**Pull request by ScalaChamp**
associated ticket:
https://broadinstitute.atlassian.net/browse/DDP-6586

### CODE REVIEW

File: src/app/participant-list/participant-list.component.ts [hunk 1]
```diff
@@ -29,6 +29,7 @@ import {Sample} from "./models/sample.model";
 import {Participant} from "./participant-list.model";
 import {FieldSettings} from "../field-settings/field-settings.model";
 import { ParticipantData } from "./models/participant-data.model";
+import { connectableObservableDescriptor } from "rxjs/observable/ConnectableObservable";
```
Comment (gmakharat) [inline src/app/participant-list/participant-list.component.ts]: unused import I think?

File: src/app/participant-page/participant-page.component.ts [hunk 2]
```diff
@@ -31,6 +31,7 @@ import { ParticipantUpdateResultDialogComponent } from "../dialogs/participant-u
 import { AddFamilyMemberComponent } from "../popups/add-family-member/add-family-member.component";
 import { Sample } from "../participant-list/models/sample.model";
 import { ParticipantDSMInformation } from "../participant-list/models/participant.model";
+import { BoundCallbackObservable } from "rxjs/observable/BoundCallbackObservable";
```
Comment (gmakharat) [inline src/app/participant-page/participant-page.component.ts]: the same I think

**Review by gmakharat — APPROVED** [review]

**Review by pegahtah — APPROVED** [review]

File: src/app/participant-list/participant-list.component.ts [hunk 3]
```diff
@@ -2100,4 +2133,20 @@ export class ParticipantListComponent implements OnInit {
     }
     return false;
   }
+
+  addAutomatedScoringColumns(): void {
+    this.showComputedObject = true;
+
+    this.dataSources.set("computed", "Morning-Evening Questionnaire Scoring");
```
Comment (SimoneMaiwald) [inline src/app/participant-list/participant-list.component.ts:2140]: Could we get these columns dynamically? If another study does scoring they would be called differently

Comment (SimoneMaiwald) [reply src/app/participant-list/participant-list.component.ts:2140]: is that information added from the DSS site into `activity_definition`?

Comment (ScalaChamp) [reply src/app/participant-list/participant-list.component.ts:2140]: @SimoneMaiwald it's added as `computed` key in ES only for `participants_structured.circadia.circadia`

Comment (SimoneMaiwald) [reply src/app/participant-list/participant-list.component.ts:2140]: so we don't have a way of getting a "nice" text/name for the keys under the computed object?

Comment (ScalaChamp) [reply src/app/participant-list/participant-list.component.ts:2140]: @SimoneMaiwald Unfortunately no, because this task is too `Circadia` - specific

Comment (SimoneMaiwald) [reply src/app/participant-list/participant-list.component.ts:2140]: ok, then lets come up with a plan when there is another study using it

**Review by SimoneMaiwald — APPROVED** [review]

**Pull request merged**